VirtoCommerce.Platform.Web/Infrastructure/PlatformOptions.cs

```diff
@@ -6,5 +6,6 @@ public class PlatformOptions
         public string DemoResetTime { get; set; }
         public string ActivationUrl { get; set; }
         public string LicenseFilePath { get; set; }
+        public string PublicKeyPath { get; set; }
```

Comment:
Need to rename to 'LicensePublicKeyPath'

Reply:
added the key